File: src/main/java/com/musinsa/subject/model/price/MinMaxPriceBrandsByCategoryNameResponse.java
package com.musinsa.subject.model.price;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
@Builder
public class MinMaxPriceBrandsByCategoryNameResponse {

    public record Brand(String brandName, String productPrice) {}

    private String categoryName;
    private List<Brand> minPriceBrands;
    private List<Brand> maxPriceBrands;

}

File: src/main/kotlin/com/musinsa/subject/model/price/MinMaxPriceBrandsByCategoryNameResponse.kt
package com.musinsa.subject.model.price

class MinMaxPriceBrandsByCategoryNameResponse(
    var categoryName: String? = null,
    var minPriceBrands: List<Brand>? = null,
    var maxPriceBrands: List<Brand>? = null
) {
    data class Brand(
        val brandName: String,
        val productPrice: String
    )
}
